rpc/core/tx.go

```diff
@@ -124,22 +109,13 @@ func TxSearch(
 	for i := skipCount; i < skipCount+pageSize; i++ {
 		r := results[i]
 
-		var proof types.TxProof
-		if proveTx {
-			block := env.BlockStore.LoadBlock(r.Height)
-			proof, err = prove.TxInclusion(consts.DefaultCodec(), block.Data, uint64(r.Index))
-			if err != nil {
-				return nil, err
-			}
-		}
```

Comment:
[non blocking comment]
We already have issues to fix this, so I think I'm okay with temporarily removing this functionality. We have to fix this functionality before creating a stable version for the audit, so I'm **also** okay with blocking until #851 is finished.

Reply:
I'd like to merge this PR to unblock universal share encoding but I can see how TxInclusion in celestia-app may be blocking for stable releases for the audit.

Reply:
true... this will also be broken as soon as we start doing anything non-interactive defaults, so perhaps we could comment out the `prove` block with a comment in the mean time?

Reply:
Can do but out of curiosity why is a commented block of code preferred over deleting it entirely? If we need to reference this code when implementing this behavior in celestia-app, the code exists in git history.

The commented code will increase cognitive load on readers of this function and may accidentally never be cleaned up if we move it to celestia-app without a corresponding PR in this repo.

Ref:
- https://kentcdodds.com/blog/please-dont-commit-commented-out-code

Reply:
good point, I'm ok with just using git history. This is still non-blocking, but I just don't like leaving an unused arg without any explanation.

Reply:
Good call, we should definitely document that the `prove` argument isn't respected